features/bidirectional-override.yml

```diff
@@ -0,0 +1,18 @@
+name: Bidirectional override
+description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm in non-HTML documents. Authors should use the `dir` global HTML attribute instead.
```

Comment:
"in non-HTML documents" doesn't seem right? Do they only apply in that case?

Reply:
They are applied in HTML documents, but use is discouraged. Stylesheets could be disabled, so text direction should be set in the HTML and not in CSS. I updated the description to clarify.